openpgm/pgm/checksum.c

```diff
@@ -88,7 +88,7 @@ static uint16_t do_csum_avx2 (const void*, uint16_t, uint32_t) PGM_GNUC_PURE;
 /* AVX-512 - Adds 512-bit operands. */
 
 static uint16_t (*do_csum) (const void*, uint16_t, uint32_t) = NULL;
-static uint32_t (*do_csumcpy) (const void* restrict src, void* restrict dst, uint16_t len, uint32_t csum) = NULL;
+static uint16_t (*do_csumcpy) (const void* restrict src, void* restrict dst, uint16_t len, uint32_t csum) = NULL;
```

Comment:
This change is to avoid a warning since all do_csumcpy functions returns an uint16_t